app/styles/app.scss

```diff
@@ -218,7 +218,7 @@ article.aria-roles {
     margin-bottom: 0.5em;
   }
 .demo-div {
-  flex: 1 1 1;
```

Comment:
Do we really anticipate traffic on IE9 or below? I'll make sure there is fallback support.